crates/next-core/src/next_manifests/client_reference_manifest.rs

```diff
@@ -151,7 +156,7 @@ impl ClientReferenceManifest {
                         .await?;
 
                     let (client_chunks_paths, client_is_async) =
-                        if let Some((client_chunks, client_availability_info)) =
+                        if let Some((client_chunks, _client_availability_info)) =
```

Comment:
Can we remove availability info here completely?

Reply:
Actually not yet. `collect_next_dynamic_chunks` still needs this tuple of chunks plus info.